.circleci/config.yml

```diff
@@ -2,13 +2,23 @@ version: 2.0
 jobs:
   build_gcc_6:
     docker:
-      - image: gpresean/anyledger-wallet
+      - image: circleci/python:3.6.6-jessie-browsers
     steps:
       - checkout
+      - run: sudo .circleci/install_deps.sh
+      - run: sudo .circleci/download_solc.sh
       - run: .circleci/run_test.sh
+  build_zephyr:
+    docker:
+      - image: mohitagnihotri/anyledger
```

Comment:
Is there a reason to use a custom image here? Otherwise please use the same image as in the other builds.